### ISSUE TITLE
Publish Dataset: native API actually uses GET not POST

### ISSUE THREAD

**Issue by bmckinney** [IQSS/dataverse#3483]
- Docs indicate POST but code specifies GET:

   @GET
    @Path("{id}/actions/:publish") 
    public Response publishDataset( @PathParam("id") String id, @QueryParam("type") String type ) {

- Inconsistent with publishing a dataverse, which does indeed use POST

**Comment by pdurbin** [comment]
@bmckinney yes, assuming this issue is a duplicate of #2431 this is something that drives me crazy. Well crazier. 😄

**Comment by pdurbin** [comment]
Fixed in pull request #3550. Passing to Code Review at https://waffle.io/IQSS/dataverse
